reladomo/src/main/java/com/gs/fw/common/mithra/cache/offheap/FastUnsafeOffHeapDataStorageJdk21.java

```diff
@@ -0,0 +1,337 @@
+/*
```

Comment:
Also not necessary for 21.
ByteBuffer has several very critical problems, which precluded its use from the start:
1) It's very slow.
2) Realloc doesn't work. This is deal breaker when allocating 100's of GB's of memory.

Those are the reasons JDK 22 introduces proper replacement for the Unsafe calls https://docs.oracle.com/en/java/javase/22/docs/api/java.base/java/lang/foreign/package-summary.html

The performance of the new memory segment stuff in 22 needs to be evaluated for replacement here.